HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,163 @@
+
+## Motivation
+
+As Holochain and the Holochain ecosystems evolves, we want a standardized process for proposing, adopting, and documenting standards of all types, including: protocol creation and changes, feature additions, and process improvements, etc. A well-defined HOW process helps:
+- Maintain transparency in decision-making
+- Ensure proper technical documentation
+- Foster community participation
+- Track the history of protocol changes
+- Enable structured discussions about significant changes
+- Enable coherent evolution of both Holochain itself, as well as the patterns and processes in use by the wider Holochain ecosystem.
+
+## Rationale
+
+This standard, as the first and the default process for adding HOWs to the HOW tree fully addresses the motivation:
+- as this process is implemented in an open github repo it both maintains transparency of the decisions made and encourages public feedback and comments in the process, as well as tracks all changes made over time.
+- the the HOW tree and the formal process for requirements for parts of the tree and adding to it ensures proper technical documentation as well as creates a clear method for evolving the process itself.
+
+## Specification
+
+### General
+HOW standards are held in a git repo on github, and use a small number of Github's affordances to help with the process.  This backing-store is intended to be replaced by a Holochain app in the future.  The HOW processes and data formats consist of a tree structure that holds all types of standards for the Holochain framework and ecosystem.
+
+### HOW Tree
+
+Rather than using the pattern of storing and identifying standards as a long numerical list (as in RFCs or EIPs), HOWs are identified by a semantic path in a hierarchy.  This allows their relationship and types to be more readily understood.  The HOW tree is stored directly as a file hierarchy in a git repo. The HOW tree encompasses different types of standards where the type is represented as the directory structure of in the tree.  The initial structure is:
+
+```
+HOW/
+├─ info/             // General guidelines or information
+├─ social/
+│  ├─ process/       // Changes to HOW processes or other organizational/ecosystem processes
+│  │  ├─ how/        // This standard
+├─ technical/
+│  ├─ core/          // Changes to core Holochain protocols
+│  ├─ application/   // Ecosystem and Application standards and conventions
+```
+
+All HOW standards MUST live as leaves of the tree.  The only files that MAY live at the branches in the tree are a `README.md` file describing the purpose of that part of the tree, and any other files related to that description, i.e. images, diagrams etc.
+
+### HOW Format
+
+HOW standards live as markdown file named `how.md` at the leaves of this directory structure.  The format of a HOW standard file consists of a markdown header and a markdown body.  The content of the header and body MUST follow a template specified by:
```

Comment:
Good point, I didn't fill out a versioning scheme.  Here a couple possiblities:

1. version goes in the file name: `process/how/how-v1.md`
2. version goes in the parent `process/how-v1/how.md/`
3. version goes in the child `process/how/v1/how.md`

In the case where the version is in the directory structure, it's possible that the how itself should allways have a canonical name like `index.md` or `how.md` or `README.md` (because the github will display it automatically when navigating the tree.

Thoughts

Reply:
I like the way it is now, because it's less overhead to click through in the filesystem / github ui.